Adding license and copyright information.

jeremiah opened the pull request:
This PR brings in some more formal licensing and copyright information into the README.

Signed-off-by: Jeremiah C. Foster <jeremiah@jeremiahfoster.com>

mardy commented:
Do we need to add our copyright line to the python files (and consequently into the README) since we have been working on the code since 2014?

jeremiah commented:
That might be a good idea. But only on the files we've changed of course.

mardy commented on README.md:
```diff
@@ -177,3 +177,12 @@ ADD_CUSTOM_COMMAND (OUTPUT ${GENERATED_STUB}
+[!License: CC BY-SA 4.0](https://img.shields.io/badge/LICENSE-CC--BY--SA--4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-nc/4.0/)
+
+
+The python code is Copyright &copy; 2008-2011 Red Hat, Inc.
```
Please add a line for Luxoft (looking at the commit history, it looks like we started working on it in 2014).

jeremiah commented:
I added the Luxoft copyright and removed Jontatan from reviewers since he seems to be on vacation. I'm going to add a 'CONTRIBUTING' file when I get a chance, likely tomorrow.

mardy commented:
Sorry, totally forgot about this, and the notification got lost :-)
